modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/wal/reader/StandaloneGridKernalContext.java

```diff
@@ -201,6 +204,7 @@ public StandaloneGridKernalContext(
         rsrcProc = new GridResourceProcessor(this);
         metricMgr = new GridMetricManager(this);
         sysViewMgr = new GridSystemViewManager(this);
+        timeoutProc = new GridTimeoutProcessor(this);
```

Comment:
It looks like it doesn't start and stop. It leads that some metrics aren't updating. See GridMetricManager#MetricsUpdater. Please add this component to the `comps` collection.